### DISCUSSION TITLE
.nosync

### DISCUSSION THREAD

**Discussion by gforti84** [abraunegg/onedrive#2462] [Q&A] [answered]
Using .nosync only works on the folder and not on the subfolders?

**Comment by abraunegg** [comment, marked as answer]
@gforti84 

> Using .nosync only works on the folder and not on the subfolders?

As per the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#how-to-skip-directories-from-syncing

The `.nosync` file works with any folder under the configured 'sync_dir' when the applicable option `--check-for-nosync` or by setting the applicable 'config' file option.

**Reply by gforti84** [reply]
Yes, that is what I read. When I was trying it yesterday, the sub-directories withing the directories I placed the .nosync file were been synced. 
Adding: 
I changed the config file to set the "check_nosync = "true"", this didn't work. Subfolders were been synced. 
Also, I got the expected results using 'sync_list' file

**Reply by abraunegg** [reply]
If you believe you have found a bug please follow the correct support process: https://github.com/abraunegg/onedrive#reporting-an-issue-or-bug

**Reply by gforti84** [reply]
I think I got it right now. I created the folders in linux only with the .nosync file. Now it shows: "Skipping downloading item - .nosync found in parent folder & ..." in verbose mode.

**Comment by NPissarra** [comment]
Hi, you can try to set skip_dir value.
https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#skip_dir
